server/events/project_command_runner.go

```diff
@@ -801,7 +802,22 @@ func (p *DefaultProjectCommandRunner) doStateRm(ctx command.ProjectContext) (out
 func (p *DefaultProjectCommandRunner) runSteps(steps []valid.Step, ctx command.ProjectContext, absPath string) ([]string, error) {
 	var outputs []string
 
-	envs := make(map[string]string)
+	// Truncate the verbose atlantis version (ex: 'dev (commit: none) (build date: unknown)' -> 'dev').
+	atlantisVersion, _, _ := strings.Cut(p.AtlantisVersion, " ")
+
+	envs := map[string]string{
+		"TF_APPEND_USER_AGENT": fmt.Sprintf(
+			"Atlantis/%s (%s; %s; %s; %s; %s; +%s)",
```

Comment:
I wonder if we should use the vcs-status-name here as the default instead of hard coded `Atlantis` since some devs use multiple atlantis instances and use that config to differentiate between them

https://www.runatlantis.io/docs/server-configuration#vcs-status-name